crates/moon/src/cli/doc.rs

```diff
@@ -52,14 +52,36 @@ pub struct DocSubcommand {
 
     #[clap(flatten)]
     pub auto_sync_flags: AutoSyncFlags,
+
+    #[clap(conflicts_with("serve"))]
+    pub symbol: Option<String>,
```

Comment:
If handy, please add the description to this positional arg as a doc comment, so it can show up in the docs.